src/page-weekly-snapshot/ImpactProjectionChart.tsx

```diff
@@ -0,0 +1,248 @@
+import { curveCatmullRom, format } from "d3";
+import * as dateFns from "date-fns";
+import React from "react";
+import { ResponsiveXYFrame } from "semiotic";
+import styled from "styled-components";
+
+import ChartWrapper from "../design-system/ChartWrapper";
+import Colors from "../design-system/Colors";
+import Loading from "../design-system/Loading";
+import { useFacilities } from "../facilities-context";
+import { useLocaleDataState } from "../locale-data-context";
+import { getChartData, ninetyDaysAgo, today } from "./projectionChartUtils";
+import { HorizontalRule } from "./SnapshotPage";
+import { useWeeklyReport } from "./weekly-report-context";
+
+const ImpactProjectionContainer = styled.div`
+  font-family: "Libre Franklin";
+  margin-bottom: 5vw;
+`;
+
+const ChartTitle = styled.h3`
+  font-size: 11px;
+  font-weight: bold;
+  line-height: 13px;
+  margin: 0 5vw;
+`;
+
+const CurveChartWrapper = styled(ChartWrapper)<{ chartHeight: number }>`
+  height: ${(props) => props.chartHeight}px;
+  display: flex;
+  flex-flow: column;
+
+  .annotation-note-title {
+    color: ${Colors.black};
+    font-family: "Libre Baskerville";
+    font-size: 40px;
+    line-height: 40px;
+    text-align: right;
+  }
+
+  .annotation-note-label {
+    font-family: "Libre Franklin";
+    font-size: 11px;
+    font-weight: 500;
+    line-height: 12px;
+    text-align: right;
+  }
+
+  .axis-title text,
+  .axis-label {
+    fill: ${Colors.black};
+    font-family: "Libre Franklin";
+  }
+`;
+
+const LegendContainer = styled.div`
+  color: ${Colors.black};
+  display: flex;
+  flex-flow: row nowrap;
+  font-size: 11px;
+  font-weight: 500;
+  margin: 0 5vw;
+`;
+
+const LegendText = styled.div<{ legendColor: string }>`
+  display: inline-block;
+  line-height: 20px;
+  letter-spacing: -0.01em;
+  margin: 0 10px;
+
+  &::before {
+    background-color: ${(props) => props.legendColor};
+    border-radius: 50%;
+    content: " ";
+    display: inline-block;
+    height: 8px;
+    margin: 0 5px;
+    width: 8px;
+  }
+`;
+
+const formatThousands = format(",~g");
+
+const lineColors: { [key in string]: string } = {
+  projectedCases: Colors.forest50,
+  projectedFatalities: Colors.black50,
+  actualCases: Colors.black,
+  actualFatalities: Colors.tamarillo,
+};
+
+const ImpactProjectionChart: React.FC = () => {
+  const {
+    data: localeDataSource,
+    loading: localeLoading,
+  } = useLocaleDataState();
+  const { state: facilitiesState } = useFacilities();
+  const {
+    state: { stateName, loading: scenarioLoading },
+  } = useWeeklyReport();
+
+  if (!stateName) return null;
+
+  const facilities = Object.values(facilitiesState.facilities);
+  const modelVersions = facilities.map((f) => f.modelVersions);
+
+  if (!scenarioLoading && !facilities.length) {
```

Comment:
Just clarifying: we don't check `localeLoading` here because it's conceptually prior to the `scenarioLoading` check, and if the local data is still loading it doesn't mean anything about the presence of scenario data. Is that right?

Reply:
Yeah, actually I don't think it makes sense to check `localeLoading` on line 214, since this component isn't going to render until locale state is loaded. I'll update this to remove that extra check